wisp-client/src/main/kotlin/wisp/client/HttpClientFactory.kt

```diff
@@ -21,7 +21,7 @@ class HttpClientFactory constructor(
     fun create(config: HttpClientEndpointConfig): OkHttpClient {
         // TODO(mmihic): Cache, proxy, etc
         val builder = unconfiguredClient.newBuilder()
-        builder.retryOnConnectionFailure(false)
+        builder.retryOnConnectionFailure(config.clientConfig.retryOnConnectionFailure ?: false)
         okHttpClientCommonConfigurator.configure(builder = builder, config = config)
```

Comment:
**suggestion (non-blocking):** I wonder if it makes sense to move this into `OkHttpClientCommonConfigurator` now?

Reply:
Thanks, I completely missed that class. Fixed